uaclient/entitlements/repo.py

```diff
@@ -77,7 +77,7 @@ def enable(self, *, silent_if_inapplicable: bool = False) -> bool:
                     print(msg)
                 util.subp(
                     ['apt-get', 'install', '--assume-yes'] + self.packages,
-                    capture=True, retry_sleeps=APT_RETRIES)
```

Comment:
What's the motivation for dropping `capture=True` here?

Reply:
actually it should be captured still. I think I took a pass at dropping that capture initially thinking that capture was what was sending a traceback to the console. That was wrong. backing it out.